src/pages/documentation/release-notes/2021.md

```diff
@@ -377,10 +292,6 @@ Please note that this release introduces breaking changes. For more details, see
 
```

Comment:
all mentions on AEP or iOS Assurance 1.x are 3.x related, we can remove from this repo